.github/CONTRIBUTING.txt

```diff
@@ -298,6 +298,31 @@ scikit-image/tools/deploy-docs.sh
 https://help.github.com/articles/creating-an-access-token-for-command-line-use/
 http://docs.travis-ci.com/user/encryption-keys/
 
+Deprecation cycle
+-----------------
+
+If the behavior of a function has to be changed, a deprecation cycle must be followed to warn users.
+Let us take an example. In version 0.12 (therefore, next release will be 0.13), we have
```

Comment:
Maybe use version ``n``, ``(n+1)`` etc. for a more general case? (Imagine when it will be scikit-image 3.42 ;-))